scroll-animations-1/Overview.bs

```diff
@@ -736,67 +736,96 @@ spec: cssom-view-1; type: dfn;
+	to the specified timeline range.
+	Delays (see 'animation-delay')
+	are set within this restricted range,
+	constricting it further.
+	If a duration ('animation-duration') is given,
+	it extends from the end of the start delay
+	until the start of the end delay,
+	and further restricts the range-restricted end.
+
+	<pre class="ascii-art">
+	range start┐             ╺┉┉┉┉active phase┉┉┉┉╸           ┌range end
+	┄┄┄┄┄┄┄┄┄┄┄├─────────────╊━━━━━━━━━━━━━━━━━━━━╉───────────┤┄┄┄┄┄┄┄┄
+               ╶┄start delay┄╴                    ╶┄end delay┄╴
+                             ╶┄┄┄┄┄ duration┄┄┄┄┄┄╴
+	</pre>
 
 	<pre class="propdef shorthand">
-		Name: animation-delay, animation-range
-		Value: [ <<'animation-delay-start'>> <<'animation-delay-end'>>? | <<timeline-range-name>> ]#
+		Name: animation-range
+		Value: [ <<'animation-range-start'>> <<'animation-range-end'>>? | <<timeline-range-name>> ]#
 	</pre>
 
-	The 'animation-delay' and 'animation-range' properties
-	are both [=shorthand property|shorthands=]
-	that set 'animation-delay-start' and 'animation-delay-end'
+	The 'animation-range' property is a [=shorthand property|shorthand=]
+	that sets 'animation-range-start' and 'animation-range-end'
 	together in a single declaration.
-	If the <<'animation-delay-end'>> value is omitted,
-	it is set to zero.
+	If the <<'animation-range-end'>> value is omitted,
+	it is set to ''normal''.
 	If a <<timeline-range-name>> alone is specified,
-	'animation-delay-start' is set to that name plus ''0%''
-	and 'animation-delay-end' is set to that name plus ''100%''.
+	'animation-range-start' is set to that name plus ''0%''
+	and 'animation-range-end' is set to that name plus ''100%''.
 
 	<pre class="propdef">
-		Name: animation-delay-start
-		Value: [ <<time>> | <<timeline-range-name>> <<percentage>> ]#
-		Initial: 0s
+		Name: animation-range-start
+		Value: [ normal | <<timeline-range-name>> <<percentage>> ]#
+		Initial: normal
 		Applies to: all elements
 		Inherited: no
 		Percentages: relative to the specified [=named timeline range=]
-		Computed value: list, each item either a duration or a timeline range and progress percentage
+		Computed value: list, each item either the keyword ''animation-range-start/normal'' or a timeline range and progress percentage
 		Animatable: no
 	</pre>
 
-	Defines a delay between the start of the animation
-	and when it begins executing
-	(i.e. when keyframes attached to 0% progress are mapped).
+	Shifts the <a spec="web-animations-1">start time</a> of the animation
+	(i.e. where keyframes mapped to 0% progress are attached
+	when the iteration count is 1)
+	to the specified position on the timeline.
 
-	<dl dfn-for="animation-delay-start, animation-delay-end" dfn-type=value>
-		<dt><dfn><<time>></dfn>
+	<dl dfn-for="animation-range-start" dfn-type=value>
+		<dt><dfn>normal</dfn>
 		<dd>
-			Defines the delay as a duration. See [[CSS-ANIMATIONS-1]].
+			The start of the [=active phase=]
+			is determined as normal.
```

Comment:
I prefer `auto` as it seems more in line with how implicitly computed defaults are specified elsewhere and aligns with `animation-duration: auto`.

I believe how we determine the start time of the animation should be updated in [web-animations-2 3.3.7. Playing an animation](https://drafts.csswg.org/web-animations/#playing-an-animation-section) to account for range start and end. This currently explains how the "normal" behavior gets a 0 start time for scroll driven animations and the current timeline time for time-based animations. The logic there should be updated to account for range start and range end and then the css-animations spec can refer to that procedure to play the animation.

Reply:
I went with `normal` because (unlike duration) we aren't magically computing a range: if the range is not otherwise defined, the timeline range is the whole timeline, as normal.

Reply:
Filed as https://github.com/w3c/csswg-drafts/issues/8406